src/main/java/net/comments/objects/BCommentManagerDuplicate.java

```diff
@@ -13,4 +22,10 @@ public void saveAndReturn(String number, String text, String state, String categ
         throw new UnsupportedOperationException("Please implement");
     }
 
+    public void saveAnReturn(String newCommentNumber) {
+        this.driver.findElement(By.xpath("//*[@id = \"Number\"]")).clear();
+        this.driver.findElement(By.xpath("//*[@id = \"Number\"]")).sendKeys(newCommentNumber);
```

Comment:
You need use separate class to work with updating comments data. `Save & Return` is only a button to save changes.